lib/core.ex

```diff
@@ -32,7 +32,7 @@ defmodule TelemetryMetricsPrometheus.Core do
     * Counter - Counter
     * Distribution - Histogram
     * LastValue - Gauge
-    * Sum - Counter
+    * Sum - Counter/Gauge
```

Comment:
I wasn't sure how best to document a new reporter option, some guidance here would be appreciated to make sure this is discoverable for folks 🙇

Reply:
Maybe you could just add an example metric definition using the option in the example above. Looks great otherwise!

Reply:
Thanks! I've added an example above and also a section outlining the currently supported `:reporter_options` further down. Let me know what you think 😄